### ISSUE TITLE
Changes made from HomeKit/Homebridge are sent to AWS but nothing actually happens on the device

### ISSUE THREAD

**Issue by rmoff** [bwp91/homebridge-govee#208]
### What issue do you have? Please be as thorough and explicit as possible.

I've set up my light strip (H6159) successfully, and it works fine directly from the Govee app. I've installed `homebridge-govee` and provided credentials. It shows my device and I can see it in Homekit. 

When I make a change in Homebridge or Homekit the light doesn't change. The log shows that the command is being sent though. If I change the light using the Govee app the log in homebridge shows it receiving the state change. 

### Details of your setup.
* Do you use (1) Homebridge UI-X (2) Homebridge CLI or (3) HOOBS? 

Homebridge UI-X

* Which version of Homebridge/HOOBS do you have?

1.4.0

* Which platform do you run Homebridge/HOOBS on (e.g. Raspberry Pi/Windows/HOOBS Box)? Please also mention your version of Node.js/NPM if known.

Docker (`oznu/homebridge:latest`)

* Which version of this plugin (homebridge-govee) do you have? Has the issue started since upgrading from a previous version?

`homebridge-govee v4.28.0`

First time I've used it.

* Which Govee devices do you have that are causing issues? Please include product models if applicable.

H6159 LED light strip

### Please paste any relevant logs below.

```
[2/8/2022, 8:04:07 AM] [Govee] Initializing Govee platform...
[2/8/2022, 8:04:07 AM] [Govee] Initialising plugin v4.28.0 | Node v16.13.2 | HB v1.4.0 | HAPNodeJS v0.10.0...
[2/8/2022, 8:04:07 AM] [Govee] Plugin initialised. Setting up accessories....
[2/8/2022, 8:04:07 AM] Homebridge v1.4.0 (HAP v0.10.0) (Homebridge FE05) is running on port 52423.
[2/8/2022, 8:04:07 AM] [Govee] Login successful.
[2/8/2022, 8:04:08 AM] [Govee] Disabling BLE client as ENODEV, No such device.
[2/8/2022, 8:04:09 AM] [Govee] [Study LED 02] if you notice any flickering when changing the brightness/colour, let me know on Discord/Github with model number [H6159].
[2/8/2022, 8:04:09 AM] [Govee] [Study LED 02] initialising with options {"adaptiveLightingShift":0,"brightnessStep":1,"disableAWS":"no","enableBT":"no","logging":"debug"}.
[2/8/2022, 8:04:09 AM] [Govee] [Study LED 02] initialised with id [XX:XX:XX:XX:XX:XX:XX:XX] [H6159].
[2/8/2022, 8:04:09 AM] [Govee] [Study LED 02] [AWS] connected event.
[2/8/2022, 8:04:09 AM] [Govee] [Study LED 02] [API] receiving update [{"source":"API","online":"false","powerState":"on","brightness":50,"color":{"r":255,"b":0,"g":255}}].
[2/8/2022, 8:04:09 AM] [Govee] [Study LED 02] has been reported [offline].
[2/8/2022, 8:04:11 AM] [Govee] ✓ Setup complete. Don't forget to ☆ this plugin on GitHub if you're finding it useful!
[2/8/2022, 8:04:24 AM] [Govee] [Study LED 02] [API] receiving update [{"source":"API","online":"false","powerState":"on","brightness":50,"color":{"r":255,"b":0,"g":255}}].
[2/8/2022, 8:04:39 AM] [Govee] [Study LED 02] [API] receiving update [{"source":"API","online":"false","powerState":"on","brightness":50,"color":{"r":255,"b":0,"g":255}}].
[2/8/2022, 8:04:43 AM] [Govee] [Study LED 02] has been reported [offline] via API, so update may or may not be successful.
[2/8/2022, 8:04:43 AM] [Govee] [Study LED 02] [AWS] sending update {"msg":{"cmd":"turn","cmdVersion":0,"data":{"val":0},"transaction":"v_1644267883930000","type":1}}.
[2/8/2022, 8:04:44 AM] [Govee] [Study LED 02] [AWS] message event [{"msg":{"cmd":"turn","cmdVersion":0,"data":{"val":0},"transaction":"v_1644267883930000","type":1}}].
[2/8/2022, 8:04:44 AM] [Govee] [Study LED 02] [AWS] receiving update [{"cmd":"turn","cmdVersion":0,"data":{"val":0},"transaction":"v_1644267883930000","type":1,"source":"AWS"}].
[2/8/2022, 8:04:44 AM] [Govee] [Study LED 02] current state [off].
[2/8/2022, 8:04:44 AM] [Govee] [Study LED 02] [API] sending update {"name":"turn","value":"off"}.
[2/8/2022, 8:04:44 AM] [Govee] [Study LED 02] [AWS] message event [{"msg":{"transaction":"o_1644267884819","type":1,"cmd":"turn","data":{"val":0}}}].
[2/8/2022, 8:04:44 AM] [Govee] [Study LED 02] [AWS] receiving update [{"transaction":"o_1644267884819","type":1,"cmd":"turn","data":{"val":0},"source":"AWS"}].
[2/8/2022, 8:04:54 AM] [Govee] [Study LED 02] [API] receiving update [{"source":"API","online":"false","powerState":"on","brightness":50,"color":{"r":255,"b":0,"g":255}}].

```

**Comment by rmoff** [comment]
Sounds similar to #131 perhaps?

**Comment by rmoff** [comment]
Per the note in #131, confirmed that if I disable Bluetooth on my phone the Govee app no longer controls the light strip either.

**Comment by rmoff** [comment]
Fixed! 
- Enable bluetooth on phone
- Connect with Govee app to light
- Confirm wifi settings (2.4Ghz network) in the app
  - I had to change iOS setting to enable privacy->location for the Govee app once for the wifi config to work
- Disable Bluetooth and confirm that Govee app can still control light 
- Celebrate!

I wonder if the homebridge-govee plugin could do something clever like look for `has been reported [offline]` condition along with state changes that are not echoed back in the status report from the API, and if those both happen flag an error pointing to the above checklist. 

anyway, all working now - thanks for your hard work on this plugin 👍
